### PR TITLE
feat(api-client): add colors to env vars and pills

### PR DESCRIPTION
**Pull request by marclave**
image

@cameronrohani need to touch this up

### CODE REVIEW

File: packages/api-client/src/components/CodeInput/codeVariableWidget.ts [hunk 1]
```diff
@@ -12,7 +12,26 @@ import {
...
+const { activeParsedEnvironments, isReadOnly, environments } = useWorkspace()
+
+const getEnvColor = (
+  item:
+    | {
+        key: string
+        value: string
+      }
+    | {
+        _scalarEnvId: any
```
Comment (amritk) [inline packages/api-client/src/components/CodeInput/codeVariableWidget.ts:24]: Dont think we need the _ here

Comment (amritk) [reply packages/api-client/src/components/CodeInput/codeVariableWidget.ts:24]: or we can... I suppose lol

**Review by amritk — CHANGES_REQUESTED** [review]
Can we factor our this duplicated function?

File: packages/api-client/src/views/Environment/EnvironmentVariableDropdown.vue [hunk 2]
```diff
@@ -41,6 +41,25 @@ const filteredVariables = computed(() => {
 const selectVariable = (variableKey: string) => {
   emit('select', variableKey)
 }
+
+const getEnvColor = (
```
Comment (amritk) [inline packages/api-client/src/views/Environment/EnvironmentVariableDropdown.vue:45]: looks familiar....

Comment (marclave) [reply packages/api-client/src/views/Environment/EnvironmentVariableDropdown.vue:45]: ohhhh yes, where should i put it?

Comment (amritk) [reply packages/api-client/src/views/Environment/EnvironmentVariableDropdown.vue:45]: libs

Comment (marclave) [reply packages/api-client/src/views/Environment/EnvironmentVariableDropdown.vue:45]: based

**Review by cameronrohani — APPROVED** [review]

**Pull request merged**